### DISCUSSION TITLE
Scheduled GitHub action failed

### DISCUSSION THREAD

**Discussion by FigBug** [community/community#26983] [Actions]
<p>I received the following error:</p>
<pre><code>Error occurred while executing servicing step &amp;#39;Disable Early Adopter customer stage lookup&amp;#39; for component CreateCollection.Framework during CreateCollection: DisableEarlyAdopterCustomerStageLookup is an invalid servicing step type for step performer FeatureAvailability during CreateCollection.
</code></pre>
<p>Any idea what it means? Looks like no steps of my workflow even tried to run.</p>

**Comment by ricardonolan** [comment]
<p>Saw same issue, and now seeing:</p>
<pre><code>The following team project collection is stopped: NQU6DH3hYFlNbTFKPwi6Mc3tUM2lsysNsZY4rpXRY2EhgYWOOi. Start the collection and then try again. Administrator Reason: Creating Collection Host
This check failed
</code></pre>

**Comment by Ferroin** [comment]
<p>Seeing the exact same here as well. I’m thinking there’s something wrong in the GitHub Actions infrastructure at the moment.</p>

**Comment by moutonjeremy** [comment]
<p>Same issue for me with 2 repositories (managed runner and self-hosted runner).</p>

**Comment by shaneiseminger** [comment]
<p>Seeing the same issue here as well. I take it to mean that it GitHub is unable to create the host the actions will run on. Nothing has changed in our action configuration, so I second/third/fourth/fifth the idea that Actions are broken.</p>

**Comment by rach1416** [comment]
<p>I am getting the same error as “This Check failed” for any github action triggered </p>

**Comment by FigBug** [comment]
<p>My action runs every hour on the hour, this time I got:</p>
<pre><code>Circuit Breaker "HttpClientThrottler-LocationHttpClient-artifactcache.actions.githubusercontent.com" short-circuited. In the last 10000 milliseconds, there were: 9 failure, 0 timeout, 21 short circuited, 0 concurrency rejected, and 0 limit rejected. Last exception: System.Threading.Tasks.TaskCanceledException A task was canceled.
</code></pre>
<p>If everybody else is having issues too I guess the problems are on GitHub’s end. However, their status page is showing everything is ok.</p>

**Comment by shaneiseminger** [comment]
<p>I was just able to get an action to run.</p>

**Comment by moutonjeremy** [comment]
<p>Same, it’s back for the moment.</p>

**Comment by ethomson** [comment, marked as answer]
<p>Apologies - this is an internal GitHub error message. We were rolling out a new deployment that had errors for some workflows, and I’m afraid that yours was one of the workflows affected.</p>
<p>We’ve rolled that deployment back - if you run your workflow now, the problem should be resolved. Apologies for the inconvenience!</p>

**Comment by chetan-hr** [comment]
<p>It seems this is happenning again for last half an hour or so but I dont see it on <a href="https://www.githubstatus.com/" rel="nofollow noopener">https://www.githubstatus.com/</a></p>
